[2022-10-04 13:30:10,734] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: agg_star_wars.create_agg_table scheduled__2022-10-03T00:00:00+00:00 [queued]>
[2022-10-04 13:30:10,749] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: agg_star_wars.create_agg_table scheduled__2022-10-03T00:00:00+00:00 [queued]>
[2022-10-04 13:30:10,750] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 13:30:10,751] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-10-04 13:30:10,751] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-10-04 13:30:10,768] {taskinstance.py:1259} INFO - Executing <Task(PostgresOperator): create_agg_table> on 2022-10-03 00:00:00+00:00
[2022-10-04 13:30:10,784] {standard_task_runner.py:52} INFO - Started process 3910 to run task
[2022-10-04 13:30:10,790] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'agg_star_wars', 'create_agg_table', 'scheduled__2022-10-03T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/star_wars_agg.py', '--cfg-path', '/tmp/tmpf2sts6n2', '--error-file', '/tmp/tmp7sqtmm4s']
[2022-10-04 13:30:10,792] {standard_task_runner.py:77} INFO - Job 46: Subtask create_agg_table
[2022-10-04 13:30:10,910] {logging_mixin.py:109} INFO - Running <TaskInstance: agg_star_wars.create_agg_table scheduled__2022-10-03T00:00:00+00:00 [running]> on host be1a5b89e146
[2022-10-04 13:30:11,116] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=xxxx@xxxxx.com
AIRFLOW_CTX_DAG_OWNER=Ram
AIRFLOW_CTX_DAG_ID=agg_star_wars
AIRFLOW_CTX_TASK_ID=create_agg_table
AIRFLOW_CTX_EXECUTION_DATE=2022-10-03T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-03T00:00:00+00:00
[2022-10-04 13:30:11,135] {base.py:79} INFO - Using connection to: id: postgres_local. Host: host.docker.internal, Port: 5432, Schema: postgres, Login: postgres, Password: ***, extra: {}
[2022-10-04 13:30:11,205] {dbapi.py:225} INFO - Running statement: CREATE TABLE IF NOT EXISTS star_wars_agg_data (
            movie_title TEXT,
            oldest_character TEXT
            ), parameters: None
[2022-10-04 13:30:11,212] {postgres.py:71} INFO - NOTICE:  relation "star_wars_agg_data" already exists, skipping

[2022-10-04 13:30:11,243] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=agg_star_wars, task_id=create_agg_table, execution_date=20221003T000000, start_date=20221004T133010, end_date=20221004T133011
[2022-10-04 13:30:11,329] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-10-04 13:30:11,386] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
